source/javascripts/filters/categories_filter.coffee

```diff
@@ -0,0 +1,3 @@
+EGA.filter 'categoriesFilter', ->
+  (items, switches) ->
+    wirsing.filter['switch'].on(items, 'category').apply switches
```

Comment:
I don't think I like the idea of using angular filters for the filtering of the awards data.
It's pretty clear that this will get out of hand very quickly -> you need to apply every filter in the html markup which is not very beautiful and I don't like logic like that in the templates